src/modules/showroom/Showroom.tsx

```
@@ -80,9 +97,11 @@ export default function Showroom() {
                 {/* Column 1: Asset Rack */}
                 <div className="w-[320px] h-full flex-shrink-0 border-r border-white/5 bg-black/40 backdrop-blur-xl">
                     <AssetRack
+                        productAsset={assetData}
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)